ironfish/src/memPool/memPool.ts

```diff
@@ -43,8 +55,14 @@ export class MemPool {
   }
 
   *get(): Generator<Transaction, void, unknown> {
-    for (const transaction of this.transactions.values()) {
-      yield transaction
+    while (!this.queue.isEmpty()) {
+      const feeAndHash = this.queue.poll()
+      if (feeAndHash) {
+        const transaction = this.transactions.get(feeAndHash.hash)
+        if (transaction) {
```

Comment:
I'd argue that this should be an assert to, because if it's falsy were ignoring a clear bug.